src/components/PageElements/PageElements.tsx

```diff
@@ -195,26 +198,28 @@ const PageElements: React.FC<PageElementProps> = ({
                   </RenderElementWrapper>
                 )
 
+              const props = {
+                applicationResponse: latestApplicationResponse,
+                reviewResponse: thisReviewLatestResponse,
+                isActiveReviewResponse: !!isActiveReviewResponse,
+                isNewApplicationResponse: !!isNewApplicationResponse,
+                isNewReviewResponse: isChangeRequest && isChanged,
+                showModal: () => updateQuery({ openResponse: element.code }),
+                summaryViewProps: summaryViewProps,
+              }
+
               return (
                 <RenderElementWrapper key={element.code}>
                   {isConsolidation ? (
                     <ConsolidateReviewDecision
-                      applicationResponse={latestApplicationResponse}
```

Comment:
not required changed

hmm ConsolidationReviewResponse doesn't have isNewReviewResponse as part of it's props, I guess it doesn't hurt passing it through, it does show why destructuring props is considered a be a bad practice. (i.e. if we remove from ReviewApplicationResponse, there wouldn't be any indication that it's unused)

Reply:
Yes, I get your point. Not sure how it would be the best practice for this either. I wonder if the `interface` defined in each component isn't enough indication of what is used in there. I was annoyed about how many same props both of them had...

Reply:
fair point, i am not sure how to deal with this either (apart from having shared props as one object, but that could be cumbersome). I guess it should be ok as long as we can see clearly what is being destructured (as it is in this case)